.travis.yml

```diff
@@ -0,0 +1,10 @@
+dist: trusty
```

Comment:
Only because I didn't want to be on the bleeding edge of travis when I started using it a few months ago.  I'll try changing to 16.04 and see what happens.  8-)